src/app/beer_garden/api/http/handlers/v1/request.py

```diff
@@ -108,6 +108,9 @@ async def patch(self, request_id):
                         raise ModelValidationError(
                             f"Unsupported status value '{op.value}'"
                         )
+                elif op.path == "/expiration_date":
+                    operation.operation_type = "REQUEST_UPDATE"
```

Comment:
You either need a second flag for setting the new date or need a different function to route to.